mp2-common/src/mpt_sequential/mod.rs

```diff
@@ -44,6 +49,11 @@ const NB_ITEMS_LEAF: usize = 2;
 /// Given we target MPT storage proof, the value is 32 bytes + 1 byte for RLP encoding.
 pub const MAX_LEAF_VALUE_LEN: usize = 33;
 
+/// This is the maximum size we allow for the value of Receipt Trie leaf
+/// currently set to be the same as we allow for a branch node in the Storage Trie
+/// minus the length of the key header and key
```

Comment:
what's the reasoning there ? To be using the same keccak gadget or stg ? Because otherwise receipt leaf size and branch node size have nothing in common.
Can you justify why it's enough ?